applications/nlp/transformer/evaluate.py

```diff
@@ -0,0 +1,282 @@
+import argparse
+import os.path
+import sys
+
+import numpy as np
+import torch
+import torch.nn
+import torchnlp.metrics
+
+# Local imports
+current_file = os.path.realpath(__file__)
+root_dir = os.path.dirname(os.path.dirname(current_file))
+sys.path.append(root_dir)
+import dataset
+import utils
+import utils.paths
+
+# ----------------------------------------------
+# Options
+# ----------------------------------------------
+
+# Hard-coded
+mini_batch_size = 64
```

Comment:
This doesn't have to match training batch size right?

Reply:
Nope. I've added a note that you don't need to match.